go.mod

```diff
@@ -59,76 +53,67 @@ require (
 	github.com/go-openapi/strfmt v0.19.3
 	github.com/go-openapi/validate v0.19.5
 	github.com/go-ozzo/ozzo-validation v3.5.0+incompatible // indirect
-	github.com/godbus/dbus v0.0.0-20181101234600-2ff6f7ffd60f // indirect
-	github.com/gogo/protobuf v1.2.2-0.20190723190241-65acae22fc9d
-	github.com/golang/groupcache v0.0.0-20160516000752-02826c3e7903
+	github.com/gogo/protobuf v1.3.1
+	github.com/golang/groupcache v0.0.0-20191227052852-215e87163ea7
 	github.com/golang/mock v1.2.0
-	github.com/google/cadvisor v0.35.0
-	github.com/google/go-cmp v0.3.0
+	github.com/google/cadvisor v0.35.1-0.20200109170520-9991c03427eb
```

Comment:
is there a tagged level we can use?

Reply:
cadvisor, we always cut a fresh release before our code freeze .. ( cc @dashpole ). Also may be better to wait until the go-winio gets sorted out first before we cut another cadvisor release

Reply:
This is fine to do for now.  I'll update it to a tagged release before the next kubernetes release.